packages/js/product-editor/src/hooks/use-blocks-helper/use-blocks-helper.ts

```diff
@@ -22,7 +22,12 @@ export function useBlocksHelper() {
 		return attributes?.id;
 	}
 
-	function getParentTabId( clientId?: string ) {
+	function getClientIdByField( field: HTMLElement ) {
+		const ancestorWithAttribute = field.closest( '[data-block]' );
+		return ancestorWithAttribute?.getAttribute( 'data-block' );
```

Comment:
The correct way of doing this is by using the [dataset](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset) prop of the element. Like:

`ancestorWithAttribute?.dataset.block`

Reply:
Done in commit [3d86563](https://github.com/woocommerce/woocommerce/pull/50035/commits/3d86563ca48e361c597ad1af17449191af914078)